### PR TITLE
Add preprocessors

### PR DESCRIPTION
**Pull request by zumkorn**
Issue #1 
I added the `createPreprocessor` function to create a preprocessor.
The developer can specify any store to listen for changes and re-render translations.
```js
import { createPreprocessor } from '@nanostores/i18n'
const screenSize = atom('big')
const sizePreprocessor = createPreprocessor(screenSize)
```
Add preprocessor:
```js
export const i18n = createI18n(locale, {
  get: …,
  preprocessors: [
    sizePreprocessor
  ]
})
```
Create messages:
```js
export const messages = i18n({
  send: sizePreprocessor.process({
    big: 'Send message',
    small: 'send'
  }),
  name: 'User name'
})
```
Return translate:
```js
export const SendLabel = () => {
  const t = useStore(messages)
  return t.send()
}
```

### CODE REVIEW

**Comment by ai** [comment]
Can you update size limit in `package.json` and add TSDoc to `createPreprocessor`?

**Comment by zumkorn** [comment]
> Can you update size limit in `package.json` and add TSDoc to `createPreprocessor`?

Added

File: README.md [hunk 1]
```diff
@@ -440,3 +440,53 @@ if (i18n.loading.get()) {
 }
 const html = ReactDOMServer.renderToString(<App />)
 ```
+### Preprocessors
```
Comment (ai) [inline README.md:443]:
```suggestion


### Preprocessors
```

**Comment by ai** [comment]
Size Limit is still invalid. You can run `pnpm test` to run all tests.

**Comment by zumkorn** [comment]
> Size Limit is still invalid. You can run `pnpm test` to run all tests.

Fixed. Sorry, my mistake.

**Comment by ai** [comment]
Looks good. Thanks.

**Pull request merged**